apache-maven/src/assembly/maven/conf/maven.properties

```diff
@@ -0,0 +1,10 @@
+#
+# Maven user properties
+#
+
+# Required files to load
+${includes} =
+
+# Optionally load global user properties and project user properties
+${optionals} = ${user.home}/.m2/maven.properties \
```

Comment:
Complete side note: We should long term reconsider getting rid of `.m2` altogether and move to a more generic terlm like `.maven`.

Reply:
That makes sense.  At least, in the short term, we can make it configurable and try to change the hardcoded references in the code to something more flexible.  I'll work on a follow-up PR, I'm already working on generating documentation for maven properties.